# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1339492575 -7200
#      Tue Jun 12 11:16:15 2012 +0200
# Branch gui
# Node ID ace446cda345a2153da6175a0588c504edd9d486
# Parent  50bf75766ac03e0617719e31736ba8539b2e8453
Changed copy-paste shortcuts from Ctrl-C/V to Ctrl-Shift-C/V, so that the Ctrl-C signal won't be catched by the terminal anymore.

* main-window.cc: Changed action shortcuts.

diff --git a/gui/src/main-window.cc b/gui/src/main-window.cc
--- a/gui/src/main-window.cc
+++ b/gui/src/main-window.cc
@@ -376,21 +376,21 @@ main_window::construct ()
 
   QMenu *edit_menu = menuBar ()->addMenu (tr ("&Edit"));
   QAction *cut_action
       = edit_menu->addAction (QIcon(":/actions/icons/editcut.png"), tr ("Cut"));
   cut_action->setShortcut (QKeySequence::Cut);
 
   QAction *copy_action
       = edit_menu->addAction (QIcon(":/actions/icons/editcopy.png"), tr ("Copy"));
-  copy_action->setShortcut (QKeySequence::Copy);
+  copy_action->setShortcut (Qt::ControlModifier + Qt::ShiftModifier + Qt::Key_C);
 
   QAction *paste_action
       = edit_menu->addAction (QIcon(":/actions/icons/editpaste.png"), tr ("Paste"));
-  paste_action->setShortcut (QKeySequence::Paste);
+  paste_action->setShortcut (Qt::ControlModifier + Qt::ShiftModifier + Qt::Key_V);
 
   QAction *undo_action
       = edit_menu->addAction (QIcon(":/actions/icons/undo.png"), tr ("Undo"));
   undo_action->setShortcut (QKeySequence::Undo);
 
   QAction *redo_action
       = edit_menu->addAction (QIcon(":/actions/icons/redo.png"), tr ("Redo"));
   redo_action->setShortcut (QKeySequence::Redo);
